papers/Gagnon_Keke_Tahiri_2024/myst.yml

```
@@ -0,0 +1,54 @@
+version: 1
+project:
+  # Update this to match `scipy-2024-<folder>` the folder should be `<firstname_surname>`
+  id: scipy-2024-00_tex_template
```

@JustGag if you update your `id` as described in the comment this should get the submission check action running as expected

---

Please update this to an ID similar to, which should match the folder name!

```suggestion
  id: scipy-2024-gagnon_keke_tahiri
```